Source/AliveExe/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ add_executable(AliveExe WIN32
     resource.rc
     resource.h
     app.ico)
+  set_target_properties(AliveExe PROPERTIES OUTPUT_NAME "relive")
```

Comment:
can't we just rename AliveExe to actually be relive instead?

Reply:
We can